gamemode/perks/arccw_shotgun_penetration.lua

```diff
@@ -0,0 +1,29 @@
+PERK.PrintName = "Buckshot Punch"
+PERK.Description = "ArcCW shotguns have {amount}mm extra penetration."
+
+PERK.Parameters = {
+    ["amount"] = {type = "i", default = 20, min = 0},
+}
+
+local recalc = function(ply, perk)
+    if perk == "arccw_shotgun_penetration" then
+        for _, wep in pairs(ply:GetWeapons()) do
```

Comment:
nit: I guess it's just a naming preference thing; I mostly use "wpn" instead of "wep" for weapons. Just for the sake of conformity, is it possible for you to refactor some of these into "wpn" as well? Thanks!